transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -134,6 +134,15 @@ public EpollEventLoopGroup(int nThreads, Executor executor, EventExecutorChooser
         super(nThreads, executor, chooserFactory, 0, selectStrategyFactory, rejectedExecutionHandler, queueFactory);
     }
 
+    public EpollEventLoopGroup(int nThreads, Executor executor, EventExecutorChooserFactory chooserFactory,
```

Comment:
Javadoc?

Reply:
Most "non-convenience" constructors lacked javadocs, so I omitted them for uniformity - will add.

Reply:
> Can we make the same changes to all `EventLoop(Group)` implementations such as `NioEventLoop(Group)`?

Yes, I will add them once approach for single implementation is accepted